### PR TITLE
feat(table-index):  show index is available or unavailable

### LINKED ISSUE #235: [Feature]:  Check the index to show whether the index is available
### Is your feature request related to a problem?

no

### Describe the solution you'd like

image
For the case where the index is not available, can the ODC not display or display the index unavailable status, otherwise the user will think that the index has been created successfully.
image

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by PeachThinking**
#### What type of PR is this?
type-feature
type-Database Object

#### What this PR does / why we need it:
Displaying whether the index is available (whether the index is created successfully) will not be used as a new column to view index information (because it is available in most cases, and it cannot be edited), but if the index is unavailable a prompt will be given to the user (similar to the prompt when viewing the PL invalid status in the object tree on the left).

Displays whether a index is available when viewing the index. If the index has been successfully created, it is in an available state, and if it has not been created successfully, it is in an unavailable state. `DBTableIndex.available` represents whether the index is available.

- Oracle judges whether it is available through the `STATUS` field of `SYS.ALL_INDEXES`.

- mysql judges whether it is available through the `COMMENT` field of `information_schema.statistics`.
mysql official documentation：
image
ob mysql official documentation：
image

#### Which issue(s) this PR fixes:
Fixes #235

#### Special notes for your reviewer:
UED：Only use hover prompts, do not add the "status" column
image

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/MySQLNoGreaterThan5740SchemaAccessor.java [hunk 1]
```diff
@@ -953,6 +955,14 @@ public List<DBTableIndex> listTableIndexes(String schemaName, String tableName)
         return new ArrayList<>(indexName2Index.values());
     }
 
+    protected void handleIndexAvailability(DBTableIndex index, String availability) {
```
Comment (yhilmare) [inline libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/MySQLNoGreaterThan5740SchemaAccessor.java:958]: how about 'enable' status?

File: libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/OBMySQLSchemaAccessor.java [hunk 2]
```diff
@@ -179,6 +179,15 @@ public List<DBTableIndex> listTableIndexes(String schemaName, String tableName)
         return indexList;
     }
 
+    @Override
+    protected void handleIndexAvailability(DBTableIndex index, String availability) {
```
Comment (yhilmare) [inline libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/OBMySQLSchemaAccessor.java:183]: duplicated code

Comment (PeachThinking) [reply libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/OBMySQLSchemaAccessor.java:183]: `OBMySQLSchemaAccessor#handleIndexAvailability`  override `MySQLNoGreaterThan5740SchemaAccessor#handleIndexAvailability`,  the implementation of these two methods is different.

File: libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/MySQLNoGreaterThan5740SchemaAccessor.java (hunk 1 above)
Comment (PeachThinking) [reply libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/schema/mysql/MySQLNoGreaterThan5740SchemaAccessor.java:958]: 👌

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**